### PR TITLE
Add kdoc about "Open in browser" from IDEA that helps to streamline working with HTML rendering

### PR DESCRIPTION
**Pull request by koperagen**
Will merge after https://github.com/Kotlin/dataframe/pull/967
Decided to add String overload `writeHtml("path.html")` to make it a little bit more convenient

### CODE REVIEW

**Review by Jolanrensen — APPROVED** [review]

File: core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/html.kt [hunk 1]
```diff
@@ -518,7 +518,11 @@ internal fun DataFrameHtmlData.print() = println(this)
 /**
  * By default, cell content is formatted as text
  * Use [RenderedContent.media] or [IMG], [IFRAME] if you need custom HTML inside a cell.
- * @return DataFrameHtmlData with table script and css definitions. Can be saved as an *.html file and displayed in the browser
+ *
+ * [DataFrameHtmlData] be saved as an *.html file and displayed in the browser.
```
Comment (Jolanrensen) [inline core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/html.kt:522]: *The [DataFrameHtmlData] will be saved...

File: core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/html.kt [hunk 2]
```diff
@@ -518,7 +518,11 @@ internal fun DataFrameHtmlData.print() = println(this)
...
+ * If you save it as a file and find it in the project tree,
+ * ["Open in browser"](https://www.jetbrains.com/help/idea/editing-html-files.html#ws_html_preview_output_procedure) feature of IntelliJ IDEA will automatically reload file content when it's updated
```
Comment (Jolanrensen) [inline core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/html.kt:524]: *, the ["Open in...

*reload the file...

File: core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/html.kt [hunk 3]
```diff
@@ -620,10 +624,18 @@ public data class DataFrameHtmlData(
...
+        File(destination).writeText(toString())
+    }
+
     public fun writeHTML(destination: Path) {
         destination.writeText(toString())
     }
 
+    /**
+     * Opens a new tab in your default browser.
+     * Consider [writeHTML] with [HTML file auto-reload](https://www.jetbrains.com/help/idea/editing-html-files.html#ws_html_preview_output_procedure) feature of IntelliJ IDEA if you want to experiment with the output and run program multiple times
```
Comment (Jolanrensen) [inline core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/html.kt:637]: *with the [HTML...

**Comment by koperagen** [comment]
@Jolanrensen Thanks :)

**Pull request merged**